MCDataProducts/inc/KalSeedMC.hh

```diff
@@ -34,15 +34,17 @@ namespace mu2e {
     MCRelationship _rel; // relationship of this particle to its primary
     uint16_t _nhits; // number of associated StrawHits
     uint16_t _nactive; // number of associated active hits
-    XYZVectorF _mom; // initial momentum
+    XYZTVectorF   _mom; // initial momentum
+    CLHEP::Hep3Vector _pos; // initial position
```

Comment:
Two things.

Please rename _pos to _startPos and _time to _startTime.  I don't know how much downstream code would need to change if I asked for the same change to _mom.  Is it prohibitiive?

@brownd1978 should she also change the space point from CLHEP to the new Root based vector classes?  Do they have distinct space and momentum vector types?

Reply:
I understand why you made the choice you did.  In my opinion the pattern should have been different and I am asking how much work it is to change the pattern - I don't know how much downstream code there is, including code that might is not covered by CI, such as TrkAna.  If it it's too much work we should not do it.  If there are may 10 places that need fixing let's do it.

Reply:
I see that Dave's opinion (in the mainline discussion, not in this thread) is that it's too disruptive.  So let's leave that alone.  

I will rerun the CI to see if the whitespace issues are all addressed.

The one item remaining is:

```
Please update CRYGenPlots to use validated fcl. We are asking everyone to do this when they touch a module.

Also please update the fetch product idiom to use the pattern from:
https://github.com/Mu2e/Tutorial/blob/main/AllInOne/src/All01_module.cc
```

Your call on whether to keep the input parameters the same or to change to the inputTag style of parameters.  If you keep the fcl parameters as is, then you can form the input tag string with something like:

( CRYInstanceName_.empty() ? CRYModuleLabel_ : CRYModuleLabel_ + ":" + CRYInstanceName_ )

In the argument to the c'tor of the input tag object.

Reply:
I dont think the CRYGenPlot module is needed for this, it was in fact an old module which I modified, should I remove it for this PR?

Reply:
You have done good work with this and it would be silly to loose it. Please convert the module to use validated fhicl.
I would like you to change the product fetch idiom too but I won't insist on it.